build/lib/pgBackRestBuild/Config/Data.pm

```diff
@@ -1251,6 +1253,16 @@ my %hConfigDefine =
         },
     },
 
+    &CFGOPT_DRYRUN =>
+    {
+       &CFGDEF_TYPE    => CFGDEF_TYPE_BOOLEAN,
+       &CFGDEF_DEFAULT => false,
+       &CFGDEF_COMMAND =>
+       {
+         &CFGCMD_EXPIRE => {},
+      },
```

Comment:
Be careful of spacing - try to make your code look like the code around it. If you use a tab character make sure your editor substitutes it with 4 spaces. Please apply this to the rest of your code in expire.c as well (see the CODING.md documentation).

Reply:
Not so simple using emacs, but I think I've done.